backend/src/main/resources/db/migrations/V1__remove_old_email_index.sql

```diff
@@ -0,0 +1 @@
+alter table administrators drop constraint administrators_email_unique;
```

Comment:
```suggestion
alter table administrators drop constraint if exists administrators_email_unique;
```

Reply:
@timoludwig If you want, you can also add your opinion to #906 :D